pkg/api/util/util.go

```diff
@@ -186,8 +217,11 @@ func InitDCAAuthToken(config model.Reader) error {
 		return nil
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
You should probably use the constant here ? (and if you do make it configurable then use the config)